internal/app/skuba/node/validate/validate_test.go

```diff
@@ -0,0 +1,52 @@
+/*
+ * Copyright (c) 2019 SUSE LLC.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ */
+
+package validate
+
+import (
+	"testing"
+)
+
+func TestNodeName(t *testing.T) {
+	testCases := []struct {
+		nodename  string
+		expectErr bool
+	}{
+		{
+			nodename:  "my-master-0",
+			expectErr: false,
+		},
+		{
+			nodename:  "my-worker-0",
+			expectErr: false,
+		},
+		{
+			nodename:  "some-too-long-hostname-foo-bar-baz-fizz-buzz-xyz-zzy-kaboom-ayy-lmao",
```

Comment:
just wanted to say I love seeing the tests come in with the original PR! nice work!